themes/socialblue/components/02-atoms/cards/cards--sky.scss

```diff
@@ -135,9 +135,11 @@
         &.block-discussion-add-block,
         &.block-group-add-event-block,
         &.block-group-add-topic-block,
+        &.block-group-add-idea-block,
         &.block-event-add-block,
         &.block-topic-add-block,
-        &.block-group-add-block {
+        &.block-group-add-block,
+        &.block-socail-idea {
```

Comment:
@agami4 There's a typo here, does the class also have a typo or does this not work?

Reply:
Actually there also was a typo in a social_idea module, but I will fix it, so @agami4, please also change strings in this PR from `.block-socail-idea` to `.block-social-idea`.

Reply:
A typo was fixed here and in the challenge module.